options.c

```diff
@@ -3717,14 +3717,35 @@ struct fio_option fio_options[FIO_MAX_OPTS] = {
 	{
 		.name	= "sync",
 		.lname	= "Synchronous I/O",
-		.type	= FIO_OPT_BOOL,
+		.type	= FIO_OPT_STR,
```

Comment:
I think this is a good way to do it, as it's backwards compatible and avoids adding a new option.